app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -742,37 +699,30 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
                 return@setOnClickListener
             }
 
-            if (!checkRequiredFields()) return@setOnClickListener
+            if (!checkRequiredFields()) {
+                rootView.snackbar(R.string.fill_required_fields_message)
```

Comment:
Have you tested it because `NestedScrollView` can host only one direct child?

Reply:
I will check it again and comment later

Reply:
Well it still shows the message here. Is it what you mean?
<img src="https://i.ibb.co/4NSQfCQ/62234966-839596133083924-7361785068756402176-n.jpg" width="300">

Reply:
Okay, It is fine then. Sometimes the app crashes when NestedScrollView is root layout and we set snackbar msg on it.